packages/twenty-server/src/database/typeorm/core/migrations/1709680520888-updateUserWorkspaceColumnConstraints.ts

```diff
@@ -25,6 +25,28 @@ export class updateUserWorkspaceColumnConstraints1709680520888
       `ALTER TABLE "core"."userWorkspace" DROP CONSTRAINT "FK_37fdc7357af701e595c5c3a9bd6"`,
     );
 
+    await queryRunner.query(`
+            DO $$
+            DECLARE
+                record RECORD;
+            BEGIN
+                FOR record IN SELECT * FROM "core"."userWorkspace"
+                              WHERE "userId" NOT IN (SELECT "id" FROM "core"."user")
+                              OR "workspaceId" NOT IN (SELECT "id" FROM "core"."workspace")
+                LOOP
+                    IF NOT EXISTS (SELECT 1 FROM "core"."user" WHERE "id" = record."userId") THEN
```

Comment:
Did you test this? I don't see any action?

Reply:
As discussed, we decided to run the following sql command instead of running on migration

`
INSERT INTO "core"."userWorkspace" ("userId", "workspaceId")
SELECT 
    u."id" AS "userId",
    u."defaultWorkspaceId" AS "workspaceId"
FROM 
    "core"."user" u
JOIN 
    "core"."workspace" w ON u."defaultWorkspaceId" = w."id"
WHERE 
    NOT EXISTS (
        SELECT 1
        FROM "core"."userWorkspace" uw
        WHERE 
            uw."userId" = u."id" AND 
            uw."workspaceId" = u."defaultWorkspaceId"
    );
`